pfcpiface/bess.go

```diff
@@ -247,6 +313,31 @@ func (b *bess) summaryLatencyJitter(uc *upfCollector, ch chan<- prometheus.Metri
 
 }
 
+func (b *bess) endMarkerSendLoop(pfcpCommAddr string, endMarkerChan chan []byte) {
+	if pfcpCommAddr == "" {
+		pfcpCommAddr = PfcpAddr
+	}
+	unixConn, err := net.Dial("unixpacket", pfcpCommAddr)
+	if err != nil {
+		log.Println("dial error:", err)
+		return
+	}
+	defer unixConn.Close()
```

Comment:
do this one time in setupinfo and store the socket in bess struct?

Reply:
done